retrofit/src/main/java/retrofit2/KotlinExtensions.kt

```diff
@@ -83,6 +84,12 @@ suspend fun <T : Any> Call<T?>.await(): T? {
   }
 }
 
+@JvmName("awaitUnit")
+suspend fun Call<Unit>.await() {
```

Comment:
If you prefer this could be made `internal` but the type resolution seems to work fine so it seems like a nice addition.